### PR TITLE
Expand tap area for speaker session

### LINKED ISSUE #386: Expand tap area for speaker session
## Kind (Required)

- Improvement

 ## Overview (Required)

- Expand tap area for speaker session
image

 ## Links	

-

### PR DESCRIPTION
**Pull request by beta1139**
## Issue
- close #386

## Overview (Required)
- Expand tap area for speaker session

## Links
-

## Screenshot
Before | After
:--: | :--:
 |

### CODE REVIEW

**Comment by roana0229** [comment]
👀

File: feature/session/src/main/res/layout/item_speaker_session.xml [hunk 1]
```diff
@@ -39,47 +39,57 @@
...
-            app:tint="?android:attr/textColorPrimary"
-            />
+            <TextView
+                android:id="@+id/speaker_session_name"
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                android:lineSpacingExtra="8sp"
+                android:text="@{speechSession.title.getByLang(lang)}"
+                android:textAppearance="@style/TextAppearance.DroidKaigi.Subtitle2"
+                app:isVisible="@{speechSession.hasSpeaker}"
```
Comment (roana0229) [inline feature/session/src/main/res/layout/item_speaker_session.xml]: Could you please change to set `isVisible` of @id/session parent view ?

Comment (beta1139) [reply feature/session/src/main/res/layout/item_speaker_session.xml]: Thanks, I fixed it.

**Comment by roana0229** [comment]
Thanks for your quick work 💯

**Pull request merged**